### ISSUE TITLE
Not possible to connect to broker after upgrade to Mosquitto 1.4.9-5 

### ISSUE THREAD

**Issue by mountaindude** [SynoCommunity/spksrc#2468]
### Setup

Mosquitto
1.4.9-5

NAS model: DS1515+
DSM version: DSM 6.0.2-8451 Update 1
### Expected behavior

The Mosquitto broker running on the NAS should work just as before upgrading to the new Mosquitto version
### Actual behavior

Not possible to connect to the Mosquitto broker. 
Have tried with various MQTT clients - no luck.
It is however possible to telnet into port 1883 of the NAS - so the server is running.
The Mosquitto service is also 
### Steps to reproduce

Upgrade to version 1.4.9-5
#### Package log

_Check Package Center or `/usr/local/{package}/var/`_
No log files written to /usr/local/mosquitto/var/
#### Other logs

_E.g. `/var/log/messages` or `/var/log/synopkg.log`_

```
Insert log here
```

**Comment by KaraokeStu** [comment]
This may be a DSM 6 issue, but I can't be sure. Unfortunately, I just upgraded the package to the latest source, so can't tell you any more about what is going on.

**Comment by KaraokeStu** [comment]
Tested using MQTT tool from iOS to my synology and works as expected on dsm 5.2

**Comment by KaraokeStu** [comment]
@mountaindude Can you confirm if a reboot fixes the issue?

**Comment by Octoate** [comment]
Hi,
I just tried to install the latest Mosquitto version from the SynoCommunity repository and it didn't work, too. It can be installed, but after trying to start the service it says that the service could not be started and I was also unable to connect to it. After I rebooted the DiskStation, the service started as expected and I am able to connect to it. At least on my system, the reboot fixed the issue.
My DSM version is "DSM 6.0.2-8451 Update 1" on a DS212j.
